lib/functions/image/partitioning.sh

```diff
@@ -42,7 +42,7 @@ function prepare_partitions() {
 	if [[ $HOSTRELEASE =~ buster|bullseye|bookworm|sid|focal|impish|hirsute|jammy|kinetic|lunar|ulyana|ulyssa|uma|una|vanessa|vera ]]; then
 		mkopts[ext4]="-q -m 2 -O ^64bit,^metadata_csum"
 	fi
-	# mkopts[fat] is empty
+	mkopts[fat]='-a'
```

Comment:
Please use `pre_prepare_partitions` hook.

Keeping current options for all other boards is a better idea.

Reply:
That works, I'll make this only for K3 boards for now.